examples/pennylane_skip_variational_classifier.py

```diff
@@ -85,17 +83,14 @@ def layer(W):
 # We use the ``BasisState`` function provided by PennyLane, which expects
```

Comment:
I cannot comment on the lines before "The quantum state $|:raw-latex:psi `:raw-latex:rangle $ after state preparation is a computational basis state that has 1s where :math:`x has 1s, for". (Are they deleted?) The maths is not displayed correctly!

Reply:
on it. Thanks!

Reply:
by the way, what do you mean by `1s` here?